Makefile

```
@@ -207,10 +207,14 @@ CGO_ENABLED := 0
 endif
 export CGO_ENABLED
 
+export GIT_REPO 		 ?= github.com/operator-framework/operator-controller
+export VERSION           ?= $(shell git describe --tags --always --abbrev=0)
```

`--abbrev=0` would be misleading for builds from non-tagged commits

```suggestion
export VERSION           ?= $(shell git describe --tags --always)
```